file_catalog/mongo.py

```diff
@@ -18,15 +19,14 @@
 
 class Mongo(object):
     """A ThreadPoolExecutor-based MongoDB client"""
-    def __init__(self, host=None, port=None):
-        kwargs = {}
-        if host:
-            kwargs['host'] = host
-        if port:
-            kwargs['port'] = port
-        logger.info('MongoClient args: %r', kwargs)
-
-        self.client = MongoClient(**kwargs).file_catalog
+    def __init__(self, host=None, port=None, username=None, password=None):
+        password_hash = (hashlib.sha256(password.encode('utf-8')).hexdigest() if password else None)
+        logger.info('MongoClient args: host=%s, port=%s, username=%s, '
+                               'sha256(password)=%s' %
+                            (host, port, username, password_hash))
```

Comment:
What's the reason for the password hash?  Is it just logging?  If so, that seems unnecessary.